pression/compressor.cpp

```diff
@@ -62,7 +62,7 @@ class Compressor : public PluginInstance
         if( name <= EQ_COMPRESSOR_NONE )
             return true;
 
-        plugin = registry.findPlugin( name );
+        plugin = pression::PluginRegistry::getInstance().findPlugin( name );
         LBASSERT( plugin );
         if( !plugin )
             return false;
```

Comment:
I can't comment on line 76, but I think you should return true or remove the assert, it doesn't make sense to have both. Also, I've checked in PluginInstance that info is EqCompressorInfo, since you're changing the API, that type should be renamed.